packages/rOpenScPCA/tests/testthat/test-calculate-clusters.R

```diff
@@ -1,9 +1,20 @@
-test_mat <- matrix(
-  runif(1000, -3, 3),
-  nrow = 100,
-  ncol = 10
+suppressPackageStartupMessages(library(SingleCellExperiment))
+
+set.seed(2024)
+sce <- splatter::simpleSimulate(nGenes = 1000)
+assay(sce, "logcounts") <- log1p(counts(sce))
+suppressWarnings({
+  sce <- scater::runPCA(sce)
+})
```

Comment:
A bit simpler maybe, since you already have `scater`

```suggestion
set.seed(2024)
sce <- splatter::simpleSimulate(nGenes = 1000, verbose = FALSE) |>
  scater::logNormCounts() |>
  scater::runPCA(ncomponents = 10)
```